src/clr-angular/forms/styles/containers.spec.ts

```diff
@@ -281,11 +281,15 @@ describe("Form layouts", () => {
         });
 
         describe("textarea", () => {
+            let textarea;
+            beforeEach(() => {
+                textarea = height("#textarea textarea");
```

Comment:
would make it slightly easier to follow to name this `textarea_height`. not a big deal though.